tests/Bridge/Laravel/Console/Commands/LogUnindexedSearchItemCommandTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Auditing\Bridge\Laravel\Console\Commands;
+
+use LoyaltyCorp\Auditing\Bridge\Laravel\Console\Commands\LogUnindexedSearchItemCommand;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Tests\LoyaltyCorp\Auditing\Stubs\Services\LogWriterStub;
+use Tests\LoyaltyCorp\Auditing\Stubs\Services\SearchLogWriterStub;
+use Tests\LoyaltyCorp\Auditing\TestCase;
+
+/**
+ * @covers \LoyaltyCorp\Auditing\Bridge\Laravel\Console\Commands\LogUnindexedSearchItemCommand
+ */
+class LogUnindexedSearchItemCommandTest extends TestCase
+{
+    /**
+     * Test command runs and returns a table
+     *
+     * @return void
+     *
+     * @throws \ReflectionException
+     */
+    public function testCommandRunsSuccessfully(): void
+    {
+        self::assertStringStartsWith('Indexing log items for search', $this->runCommand());
```

Comment:
Seems slightly weak test, especially since you've gone to all the effort of stubbing services and capturing output.

Reply:
It is a unit test to test `handle()` method of the command and not functional test hence stubbing and calling `handle()` to do its thing.

I can create functional test separately if that is what is expected.